src/frameworks/next/index.ts

```diff
@@ -303,17 +303,17 @@ export async function ɵcodegenFunctionsDirectory(sourceDir: string, destDir: st
  * Create a dev server.
  */
 export async function getDevModeHandle(
-  dir: Parameters<NonNullable<Framework['getDevModeHandle']>>[0],
-  hostingEmulatorInfo: Parameters<NonNullable<Framework['getDevModeHandle']>>[1]
+  dir: Parameters<NonNullable<Framework["getDevModeHandle"]>>[0],
```

Comment:
I think we can simplify the typings here a little bit, take on a TODO to namespace it

Reply:
Indeed. I replaced it with the explicit types required and added a TODO to reuse it from the `Frameworks` interface